src/hooks/World.py

```diff
@@ -114,6 +115,10 @@ def before_create_item(item_name: str, world: World, multiworld: MultiWorld, pla
 def after_create_item(item: ManualItem, world: World, multiworld: MultiWorld, player: int) -> ManualItem:
     return item
 
+# Use this function to change the valid filler items to be created to replace item links or starting items
+def get_filler_item_name() -> str:
+    return filler_item_name
+

```

Comment:
I think it would make more sense for this to have a default implementation in the Manual world itself, then the hook would allow overriding it like other hooks do. So we don't have core required functionality living in hooks files.

Could be a before or after, or could be a "hook" like interpret_slot_data uses.

Reply:
Ok, reworking to behave like `interpret_slot_data`

Does it make sense to leave the hook in the `World.py` file, or does it belong in the `Items.py` hook file?

Reply:
Sorry, just saw this. World.py makes more sense since it's a method on the world object, like create_item, etc. Basically, each of the hook files should provide a hook for the same named file that the method came from, so Item/Location hook files are basically just "before or after item/location table processing".

Reply:
Forgot to ping on above late comment @axxroytovu 😃

Reply:
Fixed